### ISSUE TITLE
Implement some tests for LCDFonts

### ISSUE THREAD

**Issue by ajkeller34** [PainterQubits/Devices.jl#23]
I had some ideas for tests whenever you have some time:

- [x] `linelimit`: Test for the expected bounding box for a cell given a large string input using `bounds(cell)`
- [x] `scripting`: Test bounding box for a short string for `scripting` equal to `true` or `false`
- [x] `pixelsize`, `pixelspacing`: Test expected bounding boxes for a single character string given various pixelsize, pixelspacing inputs, including w/ and w/o units?
- [x] Test output files from `referenced_characters_demo`, `characters_demo`, `scripted_demo`. One idea would be to add in `runtests.jl` the lines:
  ```
  characters_demo("/path/to/output.gds")
  @test length(read("/path/to/output.gds")) == 30254
  ```
  or whatever, where the number is the expected length of the file in bytes (which is printed when you save the file). A more sophisticated version would be to do a byte-for-byte comparison with an existing file that you commit to the `test` directory.

P.S. docs now include inline demos using SVG graphics.

**Comment by mtfang** [comment]
Implemented [here](https://github.com/PainterQubits/Devices.jl/commit/eb1d82dfb111477fa2d8a54b0dec42d426844365).

**Comment by ajkeller34** [comment]
Closed in #25
